uiowa/uiowa
# [P2LB] Turn off V2 and P2LB

Resolves https://github.com/uiowa/uiowa/issues/6814

# How to test
```
ddev blt sync --site staff-council.uiowa.edu && ddev drush @staff-council.local config-split:activate p2lb -y && ddev drush @staff-council.local uli admin/content/sitenow-converter
```

```
ddev blt sync --site faculty-senate.uiowa.edu && ddev drush @faculty-senate.local config-split:activate p2lb -y && ddev drush @faculty-senate.local uli admin/content/sitenow-converter
```

1. Convert all pages
2. Check some card link conversions related to the change here https://github.com/uiowa/uiowa/pull/6830/commits/9be1262564ebffc0bc1c7c30aeafebfe956a0bf5
3. Publish latest revisions `/admin/content?title=&type=page&status=All&field_tags_target_id=`
4. Cleanup

```
ddev drush @staff-council.local sitenow_p2lb:cleanup
```

```
ddev drush @faculty-senate.local sitenow_p2lb:cleanup
```

- Observe that paragraph types are gone (except for the v3 ones)
- Running a `cim` after the drush command should note no changes/diff.
- Inspecting the database before and after there should be ~94 less tables and upon closer inspection, things like `node__field_page_content_block` and `paragraph__field_section_content_block` should not be there.
- Browser console should say v3.

Try testing on another website (NOT gis.sites.uiowa.edu, that site has some gremlins). Also keep in mind #6859, so like for brand, deleting the other revisions newer than the protected revision needs to happen before the drush command is triggered or the config import at the end will error out.
